xarray/coding/times.py

```diff
@@ -180,6 +180,9 @@ def decode_cf_timedelta(num_timedeltas, units):
     """Given an array of numeric timedeltas in netCDF format, convert it into a
     numpy timedelta64[ns] array.
     """
+    warnings.warn('Decoding timedeltas been deprecated and '
```

Comment:
I'm not sure if you want to deprecate `decode_cf_timedelta` too or if you want to keep that around in case anyone wants to manually convert data to timedeltas. I don't have a user case for that but it was the `default` behavior, so maybe someone uses this.

Reply:
Makes sense.